src/app/modules/coding-test/components/quiz-test/quiz-test.component.ts

```diff
@@ -20,7 +20,7 @@ export class QuizTestComponent implements OnInit, OnDestroy {
    * Displays the timer
    */
   timerCount: any;
-  showError: boolean = false;
+  showError = false;
```

Comment:
It's a unused variable, removed it under https://github.com/teamclairvoyant/clairthon-rising-hackatoons/pull/34